src/main/resources/assets/tfc/recipes/gunpowder.json

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "minecraft:crafting_shapeless",
+  "ingredients": [
+    {
+      "item": "tfc:powder/saltpeter"
+    },
+    {
+      "item": "tfc:powder/saltpeter"
+    },
+    {
+      "item": "tfc:powder/sulfur"
+    },
+    {
+      "item": "tfc:powder/charcoal"
+    }
```

Comment:
*readies best clay impression*. Ore dictionary values... all of these should be ore dictionary values.

Reply:
Done, thanks!